# Stefan Teleman <stefan.teleman@oracle.com>
# Avoid build errors on Solaris because of -gtoggle.
# Create a vanilla bootstrap config file, and use it as argument
# to --with-build-config=bootstrap-O2 (Oh-2).
# Not for upstream, this is just for us.
--- /dev/null	2015-12-28 12:59:59.000000000 -0800
+++ config/bootstrap-O2.mk	2015-12-28 12:57:24.841537382 -0800
@@ -0,0 +1 @@
+BOOT_CFLAGS := -O2 $(filter-out -O%, $(BOOT_CFLAGS))
